app/migrations.go

```diff
@@ -138,8 +138,8 @@ func (a *App) DoEmojisPermissionsMigration() {
 		return
 	}
 
-	systemAdminRole.Permissions = append(systemAdminRole.Permissions, model.PERMISSION_MANAGE_EMOJIS.Id)
-	systemAdminRole.Permissions = append(systemAdminRole.Permissions, model.PERMISSION_MANAGE_OTHERS_EMOJIS.Id)
+	systemAdminRole.Permissions = append(systemAdminRole.Permissions, model.PERMISSION_CREATE_EMOJIS.Id, model.PERMISSION_DELETE_EMOJIS.Id)
+	systemAdminRole.Permissions = append(systemAdminRole.Permissions, model.PERMISSION_DELETE_OTHERS_EMOJIS.Id)
```

Comment:
So this "old style" migration isn't migrated to the new system? But it is updated so for old installs upgrading, it will skip the previous migration and go straight in with the latest current one. I wonder whether a more Django style "migrations are never changed once released" approach might be better, so we migrate to the old way, then to the latest way in the latest migration?

Reply:
This is "old-style" in the sense of migrations, but actually is not a permissions change, is a config to permission migration. I would love to have a more "Django-Style" migrations, but for that we need a proper migrations system. What I did here is to have a tool to migrate permissions deriving them from a previous permission version.

Reply:
OK got it - so this is it's own thing because it's a `config -> permissions system` migration rather than a `permissions system -> permissions system` one. Makes sense to me.